typings/tests/hook.tsx

```diff
@@ -2,14 +2,6 @@ import React from "react";
 import { useDropzone, DropzoneProps } from "../../";
 
 export const Dropzone = ({ children, ...opts }: DropzoneProps) => {
-  const { ...state } = useDropzone(opts);
-  return children(state);
+  const state = useDropzone(opts);
+  return <>{children(state)}</>;
```

Comment:
Import React from "react" is used here; if there is no <> there will be the following error
 @typescript-eslint/no-unused-vars

Reply:
I haven't seen that error so far. How did you get it?

Reply:
because I removed this useless code 🤣
<img width="1707" alt="Screen Shot 2022-04-27 at 11 20 04" src="https://user-images.githubusercontent.com/54422571/165433961-b7707619-3114-4528-9540-919f1c3ab0e8.png">

Reply:
Yeah, that code needs to be added back, then you won't get that warning.

Reply:
This is an optimization, I can delete it if you don't like it

Reply:
ok, i have reverted the code